subprojects/packagefiles/lagom/0002-CMake-Remove-directory-dependency-on-Base-res-emoji.patch

```diff
@@ -0,0 +1,25 @@
+From 9b4826f41262fa6bdf70ac4f877ab2e15f9b6995 Mon Sep 17 00:00:00 2001
+From: Andrew Kaster <andrewdkaster@gmail.com>
+Date: Wed, 2 Aug 2023 20:59:09 -0600
+Subject: [PATCH 2/2] CMake: Remove directory dependency on Base/res/emoji
+
+---
+ Meta/CMake/unicode_data.cmake | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Meta/CMake/unicode_data.cmake b/Meta/CMake/unicode_data.cmake
+index dc97c63..b701d54 100644
+--- a/Meta/CMake/unicode_data.cmake
++++ b/Meta/CMake/unicode_data.cmake
+@@ -123,7 +123,7 @@ if (ENABLE_UNICODE_DATABASE_DOWNLOAD)
+         # which only happens if files within it are added or deleted, but not when a file is modified.
+         # This is fine for this use-case, because the contents of a file changing should not affect
+         # the generated emoji.txt file.
+-        dependencies "${EMOJI_RES_PATH}" "${EMOJI_SERENITY_PATH}"
++        dependencies "${EMOJI_SERENITY_PATH}"
```

Comment:
Not sure why, but the Meson CMake consumer gets *very* confused when it sees this Directory dependency on `Base/res/emoji`. GN doesn't understand directory dependencies either, but CMake has .. .probably too much logic to make it work.